scipy/misc/doccer.py

```diff
@@ -149,7 +193,6 @@ def indentcount_lines(lines):
         return 0
     return indentno
 
-
```

Comment:
If you need to make any more changes to this function, might as well revert this to help with PEP8 / #8388. But it's not worth a separate commit unless you in a particularly generous, pedantic, and/or bored mood.

Reply:
Whoops, removing that line blank line was indeed a mistake.  I think I was doing and undoing some bulk text movement and must have removed one line too many.  I'll fix that.